docs/user-guide/kubernetes.md

```diff
@@ -330,6 +330,61 @@ echo "$(minikube ip) traefik-ui.minikube" | sudo tee -a /etc/hosts
 
 We should now be able to visit [traefik-ui.minikube](http://traefik-ui.minikube) in the browser and view the Træfik Web UI.
 
+## Basic Authentication
+
+It's possible to add additional authentication annotations in the Ingress rule. The source of the authentication is a secret that contains usernames and passwords inside the the key auth. To read about basic auth limitations see the [Kubernetes Ingress](https://docs.traefik.io/configuration/backends/kubernetes/) configuration page. 
```

Comment:
It should be possible to use a relative URL like `/configuration/backends/kubernetes`. For an example, see [here](https://raw.githubusercontent.com/containous/traefik/07c6e335980a80cc58dcbed686809e188b16dcf1/docs/configuration/backends/marathon.md) (it's the "reverse direction", though).